src/EnergyPlus/PollutionModule.cc

```diff
@@ -1898,173 +1898,173 @@ void GetPollutionFactorInput(EnergyPlusData &state)
                                     cCurrentModuleObject,
                                     "OtherFuel2",
                                     state.dataIPShortCut->cAlphaFieldNames(14),
                                     state.dataIPShortCut->cAlphaArgs(14),
-                                    Pollution.OtherFuel2Coef.NMVOCSched,
+                                    Pollution.OtherFuel2Coef.HgSched,
                                     ErrorsFound);
                 }
-                Pollution.OtherFuel2Coef.Hg = state.dataIPShortCut->rNumericArgs(14);
+                Pollution.OtherFuel2Coef.Pb = state.dataIPShortCut->rNumericArgs(14);
                 if (!state.dataIPShortCut->lAlphaFieldBlanks(15)) {
                     CheckFFSchedule(state,
                                     cCurrentModuleObject,
                                     "OtherFuel2",
                                     state.dataIPShortCut->cAlphaFieldNames(15),
                                     state.dataIPShortCut->cAlphaArgs(15),
-                                    Pollution.OtherFuel2Coef.HgSched,
+                                    Pollution.OtherFuel2Coef.PbSched,
                                     ErrorsFound);
                 }
-                Pollution.OtherFuel2Coef.Pb = state.dataIPShortCut->rNumericArgs(15);
+                Pollution.OtherFuel2Coef.Water = state.dataIPShortCut->rNumericArgs(15);
                 if (!state.dataIPShortCut->lAlphaFieldBlanks(16)) {
                     CheckFFSchedule(state,
                                     cCurrentModuleObject,
                                     "OtherFuel2",
                                     state.dataIPShortCut->cAlphaFieldNames(16),
                                     state.dataIPShortCut->cAlphaArgs(16),
-                                    Pollution.OtherFuel2Coef.PbSched,
+                                    Pollution.OtherFuel2Coef.WaterSched,
                                     ErrorsFound);
                 }
-                Pollution.OtherFuel2Coef.Water = state.dataIPShortCut->rNumericArgs(16);
+                Pollution.OtherFuel2Coef.NucHi = state.dataIPShortCut->rNumericArgs(16);
                 if (!state.dataIPShortCut->lAlphaFieldBlanks(17)) {
                     CheckFFSchedule(state,
                                     cCurrentModuleObject,
                                     "OtherFuel2",
                                     state.dataIPShortCut->cAlphaFieldNames(17),
                                     state.dataIPShortCut->cAlphaArgs(17),
-                                    Pollution.OtherFuel2Coef.WaterSched,
+                                    Pollution.OtherFuel2Coef.NucHiSched,
                                     ErrorsFound);
                 }
-                Pollution.OtherFuel2Coef.NucHi = state.dataIPShortCut->rNumericArgs(17);
+                Pollution.OtherFuel2Coef.NucLo = state.dataIPShortCut->rNumericArgs(17);
```

Comment:
Changes here seem minimal, and fine.  It looks like lots of changes, but obviously it's just removing the unused field input processing.